### PR TITLE
add new parameter LIGHT_EN_BLINKM to start blinkm lights

### PR DESCRIPTION
**Pull request by BazookaJoe1900**
This suppose to make start-up cleaner for some boards that don't uses some sensors but try do start them because they are on the common rc.sensors.
some disscution is here #12409

This pr focus the blinkm leds.

### CODE REVIEW

File: src/drivers/lights/blinkm/parameters.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * BlinkM LED
```
Comment (hamishwillee) [inline src/drivers/lights/blinkm/parameters.c]: What is a BlinkM, and what is a blinkm LED? 

Generally it is a good idea if someone can infer what a parameter is for from the parameter documentation itself.

Comment (BazookaJoe1900) [reply src/drivers/lights/blinkm/parameters.c]: @hamishwillee I must say I don't know exally what is blinkm, I don't use it.
I can only guess its this device: [blinkm](https://www.sparkfun.com/products/8579)
the code also don't specify that too good. But without more knowledge, I don't want to document that

Comment (hamishwillee) [reply src/drivers/lights/blinkm/parameters.c]: Fair enough!

**Comment by MaEtUgR** [comment]
After this pr I noticed a reoccuring error in the startup console output (seen when running `dmesg` on the MAVLink shell). There's always the error
blinkm
Because in
https://github.com/PX4/Firmware/blob/22e4e85eeb1d2688a05f5b74cfbf936539264b1e/ROMFS/px4fmu_common/init.d/rcS#L205
the parameter gets accessed even though it might very well be that `blinkm` is not compiled for the platform you're running e.g. mine.

**Pull request merged**